src/libsystemd-network/radv-internal.h

```diff
@@ -65,6 +67,9 @@
 /* Pref64 option type (RFC8781, section 4) */
 #define RADV_OPT_PREF64                           38
 
+/*  nd home-agent-lifetime (0-65520) */
```

Comment:
doubled space at the beginning.
`nd` -> `Neighbor Discovery` ?
Please add the reference to RFC.

Reply:
OK